Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -197,12 +209,21 @@ - (void)loadView
         [SignalApp.sharedApp setHomeViewController:self];
     }
 
+    ReminderView *outageView = [ReminderView
```

Comment:
A made a similar observation in another PR, but as we're getting more potentially reminder views, it seems like a (potentially empty) UIStackView would be cleaner than juggling constraints.

Reply:
Yup, done.

Reply:
Well, I used a stack view - I'm not sure replacing the constraints will simplify things.